Rover/mode.cpp

```diff
@@ -15,6 +15,7 @@ Mode::Mode() :
 
 void Mode::exit()
 {
+     g2.sailboat.clear_in_irons();
```

Comment:
If we do it on mode enter rather than exit we can keep all the sailboat stuff in one spot.

https://github.com/ArduPilot/ardupilot/blob/5e3f26744a1487c08847db92c60eb101c341a4bb/Rover/mode.cpp#L49-L50

Reply:
> If we do it on mode enter rather than exit we can keep all the sailboat stuff in one spot.
> 
> https://github.com/ArduPilot/ardupilot/blob/5e3f26744a1487c08847db92c60eb101c341a4bb/Rover/mode.cpp#L49-L50

I prefer to wipe my bottom before I exit the toilet, rather than waiting until the next time I enter it, so no don't agree.